### ISSUE TITLE
RStudio on Ubuntu 22.04 LTS, opens Chrome instead of the default web browser

### ISSUE THREAD

**Issue by jmcastagnetto** [rstudio/rstudio#14228]
### System details

    RStudio Edition : 
    RStudio Version : 
    OS Version      : 
    R Version       : 

### Describe the behavior you expected

- [ ] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [ ] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [ ] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.posit.co/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [ ] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by jmcastagnetto** [comment]
As a reference, this is the list of alternatives I got on this laptop

```
$ sudo update-alternatives --config x-www-browser
There are 2 choices for the alternative x-www-browser (providing /usr/bin/x-www-browser).

  Selection    Path                           Priority   Status
------------------------------------------------------------
  0            /usr/bin/google-chrome-stable   200       auto mode
* 1            /usr/bin/firefox                40        manual mode
  2            /usr/bin/google-chrome-stable   200       manual mode
```

And even doing the following steps, the issue remains:

1. Quitting RStudio Desktop
2. Forcing Firefox priority using: 
```
sudo update-alternatives --install /usr/bin/x-www-browser x-www-browser `which firefox` 500
```
3. Restarting RStudio and trying to open, Chrome is used even when is not the default web browser

Finally the `xdg-settings` are as follows:

```
$ xdg-settings --list
Known properties:
  default-url-scheme-handler    Default handler for URL scheme
  default-web-browser           Default web browser

$ xdg-settings get default-web-browser
firefox_firefox.desktop
```

**Comment by kevinushey** [comment]
I'm guessing this is the case given what you shared already, but just to confirm: does running `xdg-open <file>` from a terminal also open Firefox?

**Comment by jmcastagnetto** [comment]
@kevinushey Yes, it does open the HTML file in Firefox. If I got to guess, there might be something in the routines related to the file manager component that is not using the OS info.

**Comment by kevinushey** [comment]
For reference, in RStudio we handle opening of files externally here:

https://github.com/rstudio/rstudio/blob/7f5111a2043a49a466f83f836bfa67396f3a44dc/src/node/desktop/src/main/gwt-callback.ts#L128-L139

We already had to implement a workaround for Windows; perhaps we need something similar for Linux.
